tractor/_actor.py

```diff
@@ -129,22 +130,27 @@ async def _invoke(
 
     except (Exception, trio.MultiError) as err:
 
-        if not isinstance(err, trio.ClosedResourceError):
-            log.exception("Actor crashed:")
+        # TODO: maybe we'll want differnet "levels" of debugging
+        # eventualy such as ('app', 'supervisory', 'runtime') ?
+        if not isinstance(err, trio.ClosedResourceError) and (
```

Comment:
Pretty sure this won't end up biting us - it's basically only going to enter the debugger when we don't have a broken channel error or a mult-`Cancelled`